client/java/src/main/java/com/linecorp/centraldogma/client/AbstractCentralDogma.java

```diff
@@ -192,6 +199,69 @@ public <T> Watcher<T> repositoryWatcher(String projectName, String repositoryNam
                                                                 .start();
     }
 
+    @Override
+    public CompletableFuture<ImportResult> importDir(Path dir) {
+        if (dir.getNameCount() < 2) {
+            return exceptionallyCompletedFuture(new IllegalArgumentException(
+                    "Path must be <project>/<repo>[/…]: " + dir));
+        }
+        if (dir.isAbsolute()){
+            return exceptionallyCompletedFuture(
+                    new IllegalArgumentException("Only relative paths are supported: " + dir));
+        }
```

Comment:
Should we support an absolute path? It would be inconvenient if we had to move the directory to the location where `java` runs.